.travis/install-ivykis.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -x -e
+
+OLDPWD=`pwd`
```

Comment:
You don't need to save and restore the working directory in scripts / subshells:
https://askubuntu.com/questions/325944/how-to-change-a-directory-using-shell-script